webidl/manifest.webidl

```diff
@@ -1,15 +1,46 @@
 dictionary WebPublicationManifest {
-    required DOMString                         url;
-             DOMString                         lang;
-             TextDirection                     direction = "auto";
-             TextDirection                     readingProgression = "auto";
-             sequence<LocalizableString>       name;
-             DOMString                         id;
-             sequence<Contributor>             authors;
-             DOMString                         dateModified;
-             DOMString                         datePublished;
-             sequence<PublicationLink>         links;
-             sequence<PublicationLink>         readingOrder;
-             sequence<PublicationLink>         resources;
-             sequence<PublicationLink>         toc;
+    required DOMString                          url;
+    required DOMString                          type;
+
+             DOMString                          accessMode;
+             DOMString                          accessModeSufficient;
+             DOMString                          accessibilityAPI;
+             DOMString                          accessibilityControl;
+             DOMString                          accessibilityFeature;
+             DOMString                          accessibilityHazard;
+             DOMString                          accessibilitySummary;
+
+             DOMString                          id;
+
+             sequence<Person>                   artist;
+             sequence<(Person or Organization)> author;
+             sequence<Person>                   colorist;
+             sequence<(Person or Organization)> contributor;
+             sequence<(Person or Organization)> creator;
+             sequence<Person>                   editor;
+             sequence<Person>                   illustrator;
+             sequence<Person>                   letterer;
+             sequence<Person>                   penciler;
+             sequence<(Person or Organization)> publisher;
+             sequence<Person>                   readby;
+             sequence<(Person or Organization)> translator;
+
+             DOMString                          inLanguage;
+             TextDirection                      inDirection;
+
+             DOMString                          dateModified;
+             DOMString                          datePublished;
+
+             ProgressionDirection               readingProgression = "ltr";
+
+             sequence<LocalizableString>        name;
+
+             sequence<PublicationLink>          readingOrder;
+             sequence<PublicationLink>          resources;
+             sequence<PublicationLink>          links;
+
+             DOMString                          accessibilityReport;
```

Comment:
Why do we need a specific string for that? It should be already covered by `links`/`resources`.